### ISSUE TITLE
How to resolve ClassOrInterfaceType for extended types?

### ISSUE THREAD

**Issue by tamtam22** [javaparser/javasymbolsolver#150]
Hi,

I'm trying to resolve the symbol for 'extends', eg.
`public class example extends WebView {`

I can get WebView but am unable to resolve its type/symbol (i.e. android.webkit.WebView).

I tried to pass the following but it is not specified:
`SymbolReference<TypeDeclaration> symbol = JavaParserFacade.get(combinedTypeSolver).solve(((ClassOrInterfaceType) node));`

I tried `Type typeOfTheNode = JavaParserFacade.get(combinedTypeSolver).getType(node);` as well but it only gives an exception.

In what way would I be able to resolve the symbol of an 'extends' ?

**Comment by tamtam22** [comment]
I have managed to get the qualified name by using this, but I'm not sure if it is the correct way to use it:

~~~~
if (!classDeclaration.getExtendedTypes().isEmpty()) {
    String superClassName = typeSolver.getType(classDeclaration.getExtendedTypes(0));
    System.out.println(JavaParserFacade.get(combinedTypeSolver).getSymbolSolver().solveType(((ClassOrInterfaceType) node).getElementType()).getQualifiedName());
}
~~~~

Code used for parsing:
~~~~
public class WebViewGUI extends Activity { .... }
~~~~

Output for `classDeclaration.getExtendedTypes(0)` = Activity
Output from System.out.println() = android.app.Activity

**Comment by ftomassetti** [comment]
Yes, this is the correct way.

I have just released version 0.5.0 of JavaSymbolSolver and I plan to add a chapter on the JavaSymbolSolver on the [book on JavaParser](https://leanpub.com/javaparservisited) (available for free).

I realize we need more documentation

**Comment by tamtam22** [comment]
Thank you for your efforts and contributions in this project and the java-parser too, Federico!
